### PR TITLE
translation of the index page in download section

### PR DESCRIPTION
**Pull request by maugravena**
Translated download/index.md from en to pt-BR
Refs: nodejs/nodejs-pt#72

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-pt: Please have a review, thanks!

File: locale/pt-br/download/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+download: Download
+downloads:
+    headline: Downloads
+    lts: LTS
+    current: Atual
+    tagline-current: Recursos Mais Recentes
+    tagline-lts: Recomendado Para a Maioria dos Usuários
+    display-hint: Mostrar downloads para
+    intro: >
+        Baixar o código fonte do Node.js ou um instalador pré-compilado para o seu sistema, e comece a devenvolver hoje.
```
Comment (gustavohenke) [inline locale/pt-br/download/index.md]:
```suggestion
        Baixe o código fonte do Node.js ou um instalador pré-compilado para o seu sistema, e comece a desenvolver hoje.
```

File: locale/pt-br/download/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    WindowsInstaller: Instalador Windows
+    WindowsBinary: Binário para Windows
+    MacOSInstaller: Instalador macOS
+    MacOSBinary: Binário para macOS
+    LinuxBinaries: Binários para Linux
+    SourceCode: Código fonte
+additional:
+    headline: Plataformas adicionais
+    intro: >
+        Membros da comunidade Node.js mantêm compilados não oficiais do Node.js para plataformas adicionais. Note, que tais compilados não são mantidos pela equipe principal do Node.js e pode não estar ainda no mesmo nível de compilação da versão atual do Node.js.
```
Comment (gustavohenke) [inline locale/pt-br/download/index.md]:
```suggestion
        Membros da comunidade Node.js mantêm compilados não oficiais do Node.js para plataformas adicionais. Note, tais compilados não são mantidos pela equipe oficial do Node.js e podem não estar no mesmo nível de compilação da versão atual do Node.js.
```

**Review by gustavohenke — CHANGES_REQUESTED** [review]

**Comment by Drieger** [comment]
LGTM after @gustavohenke suggestions are addressed

**Comment by maugravena** [comment]
Thanks @gustavohenke

**Review by JCMais — APPROVED** [review]

**Comment by JCMais** [comment]
LGTM, thanks @maugravena

**Comment by ghost** [comment]
@gustavohenke: Is it fine with you?

**Review by gustavohenke — APPROVED** [review]
😃

**Comment by ghost** [comment]
Thanks all for your contributions!

**Pull request merged**